content/webapp/pages/exhibition.js

```diff
@@ -134,7 +135,11 @@ ExhibitionPage.getInitialProps = async ({req, query}) => {
 
     if (exhibition) {
       return {
+        type: 'website',
         title: exhibition.title,
+        imageUrl: exhibition.promoImage && convertImageUri(exhibition.promoImage.contentUrl, 800),
```

Comment:
We need to find a way to make sure this is typed and caught when we don't do it.